Scripts/sort-schemes.sh

```diff
@@ -0,0 +1,110 @@
+#!/bin/bash
+
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+# Sort the order of projects' schemes.
+# Usage: sort-schemes.sh
+
+# Alias for replacing symbols.
+aliasNewLine='@'
+aliasSpase='<space>'
+
+# Scheme counter.
+currentSchemeNumber=0;
+
+replace_number_or_hide_scheme() {
+
+    # Get params and remove aliases from schema name.
+    schemeName="${1//${aliasSpase}/ }_^#shared#^_"
+    fileName=$2
+    isHidden=$3
+
+    # Prepare the backup file.
+    backupFile="${fileName}.bak"
```

Comment:
Why exactly do we need it?

Reply:
It needs for use regex, because they don't work correctly with the new line symbols